pandas/core/window/rolling.py

```diff
@@ -1356,12 +1357,15 @@ def sum(self, *args, engine=None, engine_kwargs=None, **kwargs):
         nv.validate_window_func("sum", args, kwargs)
         if maybe_use_numba(engine):
             if self.method == "table":
-                raise NotImplementedError("method='table' is not supported.")
+                func = generate_manual_numpy_nan_agg_with_axis(np.nansum)
+            else:
+                func = np.nansum
             # Once numba supports np.nansum with axis, args will be relevant.
-            # https://github.com/numba/numba/issues/6610
-            args = () if self.method == "single" else (0,)
+            # https://github.com/numba/numba/issues/1269
```

Comment:
i would remove the comments here and consolidate with the generate_manual function for clarty

Reply:
Sure. Moved all the comments to the generate function